function.dd

```diff
@@ -806,8 +882,14 @@ $(H4 Inline Functions)
         compiler's decisions on enregistering variables.
         (There is no register keyword either.))
 
-        $(P If a $(GLINK2 expression, FunctionLiteral) is immediately called,
-        its inlining would be enforced normally.)
+    $(P However, if a D compiler has inline feature, and really need to control
```

Comment:
`has the ability to inline and control of this feature is really needed, it's possible to do so with`